packages/server/graphql/mutations/createOAuth1AuthorizeUrl.ts

```diff
@@ -55,6 +55,8 @@ export default {
         url
       }
     }
-    return standardError(new Error('Service not supported'), {tags: {service: provider.service}})
+    return standardError(new Error('Service not supported'), {
```

Comment:
technically, this is unreachable, but leaving it in in case we add new types or forget to extend one